modules/hooks.nix

```diff
@@ -1340,7 +1340,7 @@ in
                 default = "auto";
               };
 
-            config =
+            configuration =
```

Comment:
we should warn about this option rename and deprecate `relaxed`

Reply:
Sure. How do I create a warning?

Reply:
I tried adding them in 2602c35e7dca1e4d4c32b8f4b308e555948069ba but it fails with the following:
```nix
error: evaluation aborted with the following error message: 'cannot find attribute `hooks.yamllint.settings.relaxed''
```

I'm sure I'm missing something but from what I've seen in examples I used the modules correctly. Any help is appreciated.

Reply:
@domenkozar

Reply:
I fixed all warnings.